src/extensions/scratch3_mqtt/index.js

```diff
@@ -253,16 +253,15 @@ class Scratch3MQTTBlocks {
     }
 
     isConnected(){
-        if ( ! this._client ) return false;
-        return this._client.connected;
+      return this._client
```

Comment:
ternary operators can reduce functions to a single return statement

Reply:
If `this._client` is defined we execute the `? this._client.connected` statement, otherwise we return `: false`.

I've split up the statements into multiple lines, but it could have been one single line :)

`return this._client ? this._client.connected : false`